docs/modules/operation/pages/flows/setup.adoc

```diff
@@ -114,88 +114,61 @@ The value used for the lookup corresponds to the following fields from the vario
 
 [options="header, autowidth"]
 |===
-| Property | Description
-
-| `NetFlow v5` | `engineId`
-| `NetFlow v9` | `sourceId`
-| `IPFix` | `observationDomainId`
-| `SFlow` | `agent_address:sub_agent_id`
-| `BMP` | `bgpId`
+| Property     | Description
+| NetFlow v5   | engineId
+| NetFlow v9   | sourceId
+| IPFix        | observationDomainId
+| SFlow        | agent_address:sub_agent_id
+| BMP          | bgpId
 |===
 
-== Node Cache Configuration (Optional)
+== Node cache configuration
 
-By default, each flow document - if {page-component-title} knows it - is  enriched with node information.
+By default, each flow document, if {page-component-title} can match the IP address to a node, is enriched with node information.
 To reduce the number of queries to the database, the data is cached.
 
-The following cache properties are available to be set in `$\{OPENNMS_HOME/etc/org.opennms.features.flows.persistence.elastic.cfg`:
+The following cache properties are available to be set in `$OPENNMS_HOME/etc/org.opennms.features.flows.persistence.elastic.cfg`:
 
 [options="header, autowidth"]
 |===
-| Property | Description | Required | Default
-
-| `nodeCache.maximumSize`
-| The maximum size of the cache.
-| `false`
-| `1000`
-
-| `nodeCache.expireAfterWrite`
-| Number of seconds until an entry in the node cache is evicted. Set to 0 to disable eviction.
-| `false`
-| `300`
-
-| `nodeCache.recordStats`
-| Defines if cache statistics are exposed via JMX. Set to `false` to disable statistic recording.
-| `false`
-| `true`
-
+| Property                   | Default | Description
```

Comment:
Not a big deal, but for this one, we can say "The following optional cache properties ...." and then remove the "Optional" row entirely.